pkg/build/graph/analysis/bc.go

```diff
@@ -238,6 +238,22 @@ func FindPendingTags(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// getImageStreamTagSuggestion will return the appropriate marker Suggestion for when a BuildConfig is missing its input ImageStreamTag;  in particular,
+// it will determine whether or not another BuildConfig can produce the aforementioned ImageStreamTag
+func getImageStreamTagSuggestion(g osgraph.Graph, f osgraph.Namer, tagNode *imagegraph.ImageStreamTagNode, bcNode *buildgraph.BuildConfigNode) osgraph.Suggestion {
+	bcs := []string{}
+	for _, bcNode := range g.PredecessorNodesByEdgeKind(tagNode, buildedges.BuildOutputEdgeKind) {
+		bcs = append(bcs, f.ResourceName(bcNode))
+	}
+	if len(bcs) == 1 {
+		return osgraph.Suggestion(fmt.Sprintf("%s needs to be created by executing `oc start-build %s`.", f.ResourceName(tagNode), strings.Join(bcs[:], ",")))
```

Comment:
OK, I believe I honored the message/suggestion intent with the ImageStreamTag ones.  For ImageStreamImage, after going back and forth, settled on the opinion that those are a bit special case, and left the suggestions as is (and to what we agreed to previously).

Some code restructure which I at least viewed as positive subsequently emerged from the method rename you suggested, so totally cool from that perspective.

PTAL